bundles/org.openhab.binding.warmup/README.md

```diff
@@ -0,0 +1,86 @@
+
+Any Warmup 4iE device(s) must be registered at https://my.warmup.com/ prior to usage.
+
+This API is not known to be documented publicly. The binding api implementation has been derived from the implementations at https://github.com/alyc100/SmartThingsPublic/blob/master/devicetypes/alyc100/warmup-4ie.src/warmup-4ie.groovy and https://github.com/alex-0103/warmup4IE/blob/master/warmup4ie/warmup4ie.py, and enhanced by inspecting the GraphQL endpoint.
+
+## Supported Things
+
+### Bridge
+
+Credentials to my.warmup.com which allow the rest of the system to function
+
+### Room
+
+A room containing an individual Warmup 4iE device which is a WiFi connected device which controls a heating circuit. 
+The device is optimised for controlling underfloor heating (electric or hydronic), although it can also control central heating circuits.
+The device reports the temperature from one of two thermostats, either a floor temperature probe or the air temperature at the device.
+The separate temperatures do not appear to be reported through the API. It appears to be possible to configure two devices in a primary / secondary configuration, but it is not clear how this might be represented by the API and hasn't been implemented.
+
+## Discovery
+
+Once credentials are successfully added to the bridge, any rooms (devices) detected will be added as things to the inbox.
+
+## Thing Configuration
+
+### My Warmup Account
+
+| name  | type   | description                  | required | default |
+| --------- | -------- | ------------------------------ | ----------- | -------- |
+| username | String | Username for my.warmup.com | true | |
+| password | String | Password for my.warmup.com | true | |
+| refreshIntervalSec | Integer | Interval in seconds between automatic refreshes | true | 300 |
+
+### Room
+
+Rooms are configured automatically with a Serial Number on discovery, or can be added manually using the "Device Number" from the device, excluding the last 3 characters. The only supported temperature change is an override, through a default duration configured on the thing. This defaults to 60 minutes.
+
+| name  | type   | description                  | required | default |
+| --------- | -------- | ------------------------------ | ----------- | -------- |
+| serialNumber | String | Device Serial Number, excluding last 3 characters | true | |
+| overrideDurationMin | Integer | Duration of override when target temperature is changed | true | 60 |
+
+## Channels
+
+| channel  | type   | description                  | read only |
+| --------- | -------- | ------------------------------ | ----------- |
+| currentTemperature | Number:Temperature | Currently reported temperature | true |
+| targetTemperature | Number:Temperature | Target temperature | false |
+| overrideRemaining | Number:Time | Duration remaining of the configured override| true |
+| runMode | String | Current operating mode of the thermostat | true |
```

Comment:
Can you mention these modes?

Reply:
I've added further explanation (based on my limited knowledge!).